app.js

```diff
@@ -252,6 +254,40 @@ function resolveHeroNameToId(name, callback) {
     })
 }
 
+function sortByKey(array, key) {
+    return array.sort(function(a, b) {
+        var x = a[key]; var y = b[key];
+        return ((x > y) ? -1 : ((x < y) ? 1 : 0));
+    });
+}
+
+function promiseRequest(url) {
+    return new Promise(function (resolve, reject) {
+        request(url, function (error, res, body) {
+            if (!error && res.statusCode === 200) {
+                resolve(body);
+            } else {
+                reject(error);
+            }
+        });
+    });
+}
+
+async function getStats(userId, heroId) {
+    var statsUrl = 'https://api.opendota.com/api/players/' + userId + '/wl?hero_id=' + heroId;
+    var stats = await promiseRequest({url: statsUrl, json: true});
+    var a = 3;
+    var b = 7;

```

Comment:
basically if game outcomes follow Binomial(win_percentage), then powerrank sets a Beta(a, b) prior for win_percentage

not sure what the "objectively best" way to choose a and b is so maybe this should be debated further